### PR TITLE
New clothing 

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request
More clothing

## Why It's Good For The Game
More options to wear

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

imagen
imagen
imagen
imagen
imagen
imagen
imagen
imagen
imagen

https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/2745e91a-a42b-4406-8e00-e97e57121478

imagen

https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/a87b6d2d-c45c-4e92-87d4-f6d5524905c2

https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/ec606af9-8d6b-461b-8f48-aac718b045b5

</details>

Spur sound source: https://freesound.org/people/CGEffex/sounds/98816/
Jester shoes sound: https://freesound.org/people/Natty23/sounds/411747/
Quack sound: https://freesound.org/people/Mikes-MultiMedia/sounds/418509/

## Changelog
:cl: ClownMoff, dieamond13, kugamo, DeerJesus, san7890, ExcessiveUseOfCobblestone
add: Adds an underground Serpents Jacket. That's us, and we rule!
add: Adds a single pair of gar glasses to the contraband section of the AutoDrobe.
add: Adds the rest of the mime outfit to the spare mime outfit in the Autodrobe
add: Adds a rainbow bowtie from /TG/
add: Adds a propeller hat from /TG/
add: Adds a beige and a white fedora
add: Adds a fishing hat from /TG/, avaiable in BeeCoin shop for 10.000 coins and in the collectable hats crate
add: Adds the jester shoes  from /TG/ for the jester outfit, they jingle when you walk!
add: Adds cowboy boots, white cowboy boots, black cowboy boots, fancy cowboy boots from /TG/. They spur when walking!
add: Adds ducky Shoes from /TG/ (Originally from Paradise) to the contraband section of the AutoDrobe. They Quack when walking!
remove: Removes one of the pair of garb glasses from the contrabard part of the AutoDrobe.

/:cl:

### CODE REVIEW

**Comment by ClownMoff** [comment]
The mapping label was a mistake hihi

**Comment by ClownMoff** [comment]
"I have class at 2 am", am I an idiot? I meant 12pm

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -210,6 +210,39 @@
...
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/small
+
+/obj/item/clothing/head/fedora_beige/suicide_act(mob/living/user)
+	if(user.gender == FEMALE)
+		return
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is donning [src]! It looks like [user.p_theyre()] trying to be nice to girls.</span>")
+	user.say("M'lady.", forced = "fedora suicide")
+	sleep(10)
+	H.facial_hair_style = "Neckbeard"
```
Comment (PestoVerde322) [inline code/modules/clothing/head/misc.dm]: Make it a subtipe of Fedora.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -377,3 +377,65 @@
...
+	desc = "I got boots, that go *quack quack quack quack quack."
+	icon_state = "ducky_shoes"
+	var/footstep = 1
+
+/obj/item/clothing/shoes/ducky_shoes/step_action()
+	if(footstep > 1)
+		playsound(src, "sound/effects/quack.ogg", 50, 1)
+		footstep = 0
+	else
+		footstep++
```
Comment (PestoVerde322) [inline code/modules/clothing/shoes/miscellaneous.dm:343]: This should be under either /obj/item/clothing/shoes/clown_shoes or /obj/item/clothing/shoes/cluwne and make the sound source a var, both are horribly cursed codewise anyway.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -1104,6 +1104,12 @@
 	togglename = "zipper"
 	body_parts_covered = CHEST|GROIN|ARMS
 
+/obj/item/clothing/suit/underground_serpents_jacket // https://www.youtube.com/watch?v=S0ximxe4XtU&ab_channel=ElevatedLevel
```
Comment (PestoVerde322) [inline code/modules/clothing/suits/miscellaneous.dm]: Unnecessary long youtube link, while fine, anything post the & is not necessary.

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 2 above)
Comment (PestoVerde322) [reply code/modules/clothing/shoes/miscellaneous.dm:343]: as in both the ducky shoes and the jesters, also the Jester shoes already exists under /obj/item/clothing/shoes/clown_shoes/jester

**Review by PestoVerde322 — CHANGES_REQUESTED** [review]

**Review by PestoVerde322 — APPROVED** [review]

**Pull request closed without merging**